---
layout: default
title: Research
---
We use tools derived from field biology, molecular immunology, and epidemiological modeling to study how zoonotic pathogens persist, transmit, and evolve in wild reservoir and spillover human hosts. Much of our work is focused on zoonotic infections derived from bat reservoirs, and we conduct the majority of our field studies in Madagascar. Major areas of research in the lab are summarized below. 

<h2>Bats as reservoirs for emerging viral zoonoses</h2>

<img src="/research/flying_bat.png" alt="bat" class="img-left-w-text" />

Bats are  reservoir hosts for the world's most virulent emerging viral zoonoses, including Hendra and Nipah henipaviruses, Ebola and Marburg filoviruses, and SARS, MERS, and SARS-CoV-2 coronaviruses. The Brook Lab investigates how these pathogens which manifest as highly virulent in humans are maintained at the **population-level** in wild bat reservoir hosts. Classic epidemiological modeling explores the dynamics of perfectly immunizing childhood infections, under which hosts are  born susceptible (S), become infectious (I), and recover (R) via immune responses retained for life. In the simplest example, infections persist maintained in a population by constant re-supply of susceptible births. Many bat species reproduce in annual or biannual birth pulses, meaning that susceptible re-supply is restricted within a year. Such a system suggests that bat viruses may be governed by more nuanced dynamics than standard SIR--be they longer infectious periods, latent periods for persistent infections, or periodic waning immunity. To elucidate these dynamics, we fit candidate mechanistic models to age-structured prevalence and seroprevalence data, which we collect ourselves through the field studies of the inimitable [Ekipa Fanihy](team.html). This work is funded through an [NIH capacity-building grant](http://grantome.com/grant/NIH/R01-AI129822-01) in partnership with the [Institut Pasteur of Madagascar](http://www.pasteur.mg/) (IPM).

<div style="clear:both;">&nbsp;</div>

<ul class="citation">
  
  <li>Kettenburg G, Kistler A, Ranaivoson HC, Ahyong V, Andrianiaina A, Andry S, DeRisi JL, Gentles A, Raharinosy V, Randriambolamanantsoa TH, Ravelomanantsoa NAF, Tato CM, Dussart P, Heraud JM, and <strong>Brook CE</strong>. Full genome <em>Nobecovirus</em> sequences from Malagasy fruit bats define a unique evolutionary history for this coronavirus clade. 2021. <em> bioRxiv. </em> doi:<a href="https://www.biorxiv.org/content/10.1101/2021.09.29.462406v1">10.1101/2021.09.29.462406</a>.</li>
	<li>Ravelomanantsoa NAF, Guth S, Andrianiaina A, Andry S, Gentles A, Ranaivoson HC, and <strong>Brook CE</strong>. 2020. The zoonotic potential of bat-borne coronaviruses. <em>Emerging Topics in Life Sciences.</em> doi:<a href="https://doi.org/10.1042/ETLS20200097">10.1042/ETLS20200097</a>.</li>
	<li>Gentles A, Guth S, Rozins C, and  <strong>Brook CE</strong>. 2020. A review of mechanistic models of viral dynamics in bat reservoirs for zoonotic disease. 2020. <em>Pathogens and Global Health.</em> doi:<a href="https://doi.org/10.1080/20477724.2020.1833161">10.1080/20477724.2020.1833161</a>.</li>
	<li><strong>Brook CE</strong>, Ranaivoson HC, Broder CC, Cunningham AA, Héraud JM, Peel AJ, Gibson L, Wood JLN, Metcalf CJE*, and Dobson AP*. 2019. Disentangling serology to elucidate henipa- and filovirus transmission in Madagascar fruit bats. <em>Journal of Animal Ecology.</em> doi: <a href="https://doi.org/10.1111/1365-2656.12985">10.1111/1365-2656.12985</a>. *=equal senior contributions.</li>
	<li><strong>Brook CE</strong> and Dobson AP. 2015. Bats as ‘special’ reservoirs for emerging zoonotic pathogens. *Trends in Microbiology.* doi: <a href="https://doi.org/10.1016/j.tim.2014.12.004">10.1016/j.tim.2014.12.004</a>.</li>
</ul>

<h2>The coevolution of bats and viruses</h2>

<img src="/research/PaKi_EBOV_induced_10K_200hrs_CELL.gif" alt="bat_cell" class="img-left-w-text" />

Bats appear to host otherwise virulent viral infections without themselves experiencing substantial pathology. Accumulating molecular evidence links the evolution of flight to bat viral tolerance: unique anti-inflammatory pathways in bat immune systems enable them to avoid oxidative damage accrued during the metabolically expensive process of flight, with cascading consequences for bat longevity and resilience to intracellular infections. Bats' abilities to host viruses without disease may provide selective pressure for the evolution of pathogens which are not virulent to their bat hosts but cause extreme disease uponn spillover to non-bat (i.e. human) hosts. We use a combination of experiments carried out in bat cell tissue culture and theoretical models to investigate the co-evolution of bat immune systems and their pathogens. On average, we have shown that bat-borne zoonoses cause higher human case fatality rates than do viruses derived from any other mammalian group.

<div style="clear:both;">&nbsp;</div>

<ul class="citation">

	<li><strong>Brook CE</strong>, Rozins C, Guth S, Boots M. 2021. <em>bioRxiv</em>. Reservoir host immunology and life history shape virulence evolution in zoonotic viruses. doi (pre-print): <a href="https://www.biorxiv.org/content/10.1101/2021.10.06.463372v1">10.1101/2021.10.06.463372</a>.</li>
	<li>Guth S, Mollentze N, Renault K, Streicker DG, Visher E, Boots M, and <strong>Brook CE</strong>. 2021. <em>bioRxiv</em>. Bats host the most virulent—but not the most dangerous—zoonotic viruses. doi (pre-print): <a href="https://www.biorxiv.org/content/10.1101/2021.07.25.453574v1">10.1101/2021.07.25.453574</a>.</li>
	<li><strong>Brook CE</strong>, Boots M, Chandran KC, Dobson AP, Drosten C, Graham AL, Grenfell BT, Müller MA, Ng M, Wang LF, and van Leeuwen A. 2020. Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence. <em>eLife</em>. doi: <a href="https://elifesciences.org/articles/48401">10.7554/eLife.48401</a>.</li>
	<li>Guth S, Visher E, Boots M, and <strong>Brook CE.</strong> 2019. Host phylogenetic distance drives trends in virus virulence and transmissibility across the animal-human interface. <em>Philosophical Transactions of the Royal Society.</em> 374(1782): 20190296. doi: <a href="https://doi.org/10.1098/rstb.2019.0296">10.1098/rstb.2019.0296</a>.</li>
</ul>

<h2>Conservation biology and population modeling</h2>

<img src="/research/baby_bat.png" alt="baby_bat" class="img-left-w-text" />

Using age data derived from fruit bat dentition, we have undertaken population modeling studies for wild, Madagascar fruit bats. These studies suggest that the Madagascar Flying Fox, *Pteropus rufus*, is particularly threatened by habitat destruction and human hunting for food consumption in Madagascar. We are currently working with a local conservation NGO, [Madigasikara Voakajy](https://www.madagasikara-voakajy.org/), to develop conservation-oriented intervention plans for this species and undertaking new studies aimed at quantifying the positive benefits that this bat offers to healthy ecosystems, including pollination of several endangered Malagasy baobab species. We are also involved in several other conservation collaborations, through which we have undertaken population modeling for several non-Chiropteran threatened taxa, including endemic Malagasy lemurs and tenrecs.

<div style="clear:both;">&nbsp;</div>

<ul class="citation">

<li>Andrianiaina A, Andry S, Gentles A, Guth S, Héraud JM, Ranaivoson HC, Ravelomanantsoa
NAF, Treuer T, and <strong>Brook CE</strong>. Reproduction, seasonal morphology, and juvenile growth in three Malagasy fruit bats. 2021. <em> bioRxiv </em>. doi:
	<a href="https://biorxiv.org/cgi/content/short/2021.10.28.466299v1">10.1101/2021.10.28.466299</a>. </li>
	<li>Annapragada A, <strong>Brook CE</strong>, Luskin MS, Rahariniaina RP, Helin M, Razafinarivo O, Ralaiarison AR, Randriamady HJ, Olson LE, Goodman SM, Golden CD. Evaluation of tenrec population viability and potential sustainable management under hunting pressure in northeastern Madagascar. 2021. <em>Animal Conservation</em>. doi:
	<a href="https://doi.org/10.1111/acv.12714">10.1111/acv.12714</a>. </li>
	<li>Rocha R, Aziz SA, <strong>Brook CE</strong>, Carvalho WD, Cooper-Bohannon R, Frick WF, Huang JCC, Kingston T, Lopez-Baucells A, Maas B, Mathews F, Medellin RA, Olival KJ, Peel AJ, Plowright RK, Razgour O, Rebelo H, Rodrigues L, Rossiter SJ, Russo D, Straka TM, Teeling EC, Treuer T, Voigt CC, and  Webala PW. Bat conservation and zoonotic disease risk: A research agenda to prevent misguided persecution in the aftermath of COVID-19. 2020. <em>Animal Conservation.</em> doi:
	<a href="https://doi.org/10.1111/acv.12636">10.1111/acv.12636</a>. </li>
	<li><strong>Brook CE</strong>, Ranaivoson HC, Andriafidison D, Ralisata M, Razafimanahaka J, Héraud JM, Dobson AP, and Metcalf CJE. 2019. Population trends for two Malagasy fruit bats. Biological Conservation. 234: 165-171. doi: <a href="https://doi.org/10.1016/j.biocon.2019.03.032">10.1016/j.biocon.2019.03.032</a>. </li>
	<li><strong>Brook CE</strong>, Herrera JP, Borgerson C, Fuller E, Andriamahazoarivosoa P, Rasolofoniaina BJR, Randrianasolo JLRR, Rakotondrafarasata ZRE, Randriamady HJ, Dobson AP, and Golden CD. 2018. Population viability and harvest sustainability for Madagascar lemurs. <em>Conservation Biology.</em> doi:  <a href="https://doi.org/10.1111/cobi.13151">10.1111/cobi.13151</a>. </li>
</ul>

<h2>Next Generation Sequencing for Public Health in Madagascar</h2>

<img src="/research/christian_CZB.jpg" alt="chris_czb" class="img-left-w-text" />

Our lab was  awarded a [Bill & Melinda Gates Foundation Grand Challenges Explorations grant](https://gcgh.grandchallenges.org/grant/metagenomics-and-etiology-zoonotic-disease-deciphering-bat-human-viral-transmission-madagascar) for ‘Metagenomic Next Generation Sequencing for Pathogen Identification in Low-Income Countries.’ In part with this award, Malagasy colleagues from the [IPM](http://www.pasteur.mg/) trained in library preparation and metagenomic Next Generation Sequencing at Chan Zuckerberg Biohub (San Francisco, CA) in July 2019. In March 2020, we received an Illumina sequencing machine at IPM and, with additional support from the UC Berkeley [Innovative Genomics Institute](https://innovativegenomics.org/projects/covid-19-in-madagascar-next-generation-sequencing/), we have been using it to carry out genomic surveillance for SARS-CoV-2 variants of concern in Madagascar. In the past year, we have contributed over 700 genomes to the global genome repository, GISAID. In the future, we hope to use the iSeq to carry out concurrent  sequencing of bat excretia from our longitudinal study along with sequencing of samples from undiagnosed febrile human patients reporting to public hospitals across Madagascar.

<div style="clear:both;">&nbsp;</div>

<ul class="citation">
	<li>Andriamandimby SF<sup>*</sup>, <strong>Brook CE<sup>*</sup></strong>, Razanajatovo N, Rakotondramanga J-M, Rasambainarivo F, Raharimanga V, Razanajatovo IM, Mangahasimbola R, Razafindratsimandresy R, Randrianarisoa S, Bernardson B, Rabarison JH, Randrianarisoa M, Nasolo FS, Rabetombosoa RM, Randremanana R±, Heraud J-M±, Dussart P±. Cross-sectional cycle threshold values reflect epidemic dynamics of COVID-19 in Madagascar. In Review. <em>Epidemics.</em> doi (pre-print): <a href="https://pubmed.ncbi.nlm.nih.gov/34268517/">10.1101/2021.07.06.21259473</a>.</li>
	<li>Randremanana R, Andriamandimby SF, Rakotondramanga J-M, Razanajatovo N, Mangahasimbola R, Randriambolamanantsoa T, Ranaivoson HC, Rabemananjara H, Razanajatovo I, Razafindratsimandresy R, Rabarison J, <strong>Brook CE</strong>, Rakotomanana F, Rabetombosoa R, Razafimanjato H, Ahyong V, Raharinosy V, Raharimanga V, Raharinantoanina S, Randrianarisoa M, Bernardson B, Randrianasolo L, Randriamampionona L, Tato CM, DeRisi JR, Dussart P, Vololoniaina M, Randriatsarafara F, Randriamanantany Z, and Heraud J-M. The COVID-19 Epidemic in Madagascar: clinical description and laboratory results of the first wave, March-September 2020. <em>Influenza and Other Respiratory Viruses.</em> 2021. doi (pre-print): <a href="https://doi.org/10.22541/au.161088504.46456502/v1">10.22541/au.161088504.46456502/v1</a>.</li>
</ul>
