---
title: A simple breakdown of bat coronavirus molecular biology
layout: default
---


<font size="4">

<b>A simple breakdown of bat coronavirus molecular biology</b>

</font>

by Fifi Ravelomanantsoa

<br>
Wondering how coronaviruses made it from bats to humans?


<br>
<img src="/assets/img/Fifi_CoV.jpg" alt="fifi-logo" style="height: 260px; padding-right: 10px;" align="left">
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>

Then this blog digest of [our recent paper](https://doi.org/10.1042/ETLS20200097) is made for you!

Coronaviruses are RNA viruses notorious for their ability to switch host species—for example, to transmit from bat to human. Bats are the original source hosts for all Alpha- and Betacoronaviruses, including five of the seven coronaviruses known to infect humans: HCoV-229E, HCoV-NL63 (which cause the common cold), as well as the more deadly SARS-CoV, MERS-CoV, and SARS-CoV-2.

To “host-switch”, a virus must, first, enter the cells of a new host, and second, adapt to this new host environment. CoVs are uniquely well-suited to entering the cells of many mammals due to their flexible use of a variety of host cell surface receptors that are found on the cells of many species. For example, HCoV-229E uses the hAPN cellular receptor to enter human cells, while HCoV- NL63 uses angiotensin converting esterase (ACE2), and MERS-CoV uses dipeptidyl peptidase 4 (DPP4). All of these cell surface receptors can be found on the cells of many mammalian species—meaning that a virus capable of using these receptors in one host (i.e. bats) will be well-poised to transmit to other mammal hosts, too. Indeed, scientists believe that SARS-CoV-2-like viruses circulating in wild bats needed very few new adaptations (if any) to be able to transmit among human hosts.

Once entered into a host cell, CoVs also demonstrate remarkable adaptive capacity, resulting from their rapid mutation rates (as are common to all RNA viruses), as well as their large genome sizes and propensity for homologous recombination (traits specific to coronaviruses). CoVs are able to maintain large genomes because they encode a proofreading capacity in the RNA polymerase that they use to replicate, which helps them avoid accumulating large quantities of deleterious mutational errors. In turn, having larger genomes means that these viruses have more genetic material available for recombination—or the exchange of gene fragments between two viruses in the same cell. Recombination events help viruses switch hosts by allowing viruses to make large genetic changes very rapidly.

Among coronaviruses, the Sarbecovirus subgenus (belonging to the Betacoronavirus genus) poses a particularly dangerous pandemic threat, as has been already realized in the case of SARS-CoV and SARS-CoV-2, which fall within this clade. Sarbecoviruses are typically hosted by the Rhinolophidae horseshoe bats of Asia, Africa and Europe, including the two most closely related viruses to SARS-CoV-2, which have been previously described in Rhinolophus afﬁnis and Rhinolophus malayanus. Many genetically distinct Sarbecoviruses are known to circulent in several species of Chinese horseshoe bat that all reside in the same cave systems, thus facilitating what is known as “superinfection”, or one host’s infection with multiple different viruses at once. Bat superinfection with multiple Sarbecoviruses in turn facilitates the recombination events that allow this clade to make sudden and dramatic host shifts.

Most Sarbecoviruses that have been previously described are found in Asia, though recent work indicates that these viruses do circulate in Africa and Europe. The extent to which novel Sarbecoviruses are capable of invading human cells has not been thoroughly studied. More extensive longitudinal surveillance of coronaviruses in their native bat species, including more widespread sampling in understudied regions—like Africa (and including Madagascar!)—will be important for preventing the emergence of additional coronavirus zoonoses in the future. Much work lies ahead for Ekipa Fanihy!
